### DISCUSSION TITLE
How to Freeze the Whole Backbone

### DISCUSSION THREAD

**Discussion by weikai520** [open-mmlab/mmyolo#213] [General]
### What is the problem this feature will solve?

I think there are parameters that can freeze a layer or - 1 means that all layers are not frozen, and the whole backbone network is not frozen

### What is the feature you are proposing to solve the problem?

I think there are parameters that can freeze a layer or - 1 means that all layers are not frozen, and the whole backbone network is not frozen

**Comment by RangeKing** [comment]
For the P5 model, setting `frozen_stages=4` can freeze the whole backbone.
You can refer to the source code, 
https://github.com/open-mmlab/mmyolo/blob/c037ac8907d701809c424e14cecc4529308969d1/mmyolo/models/backbones/base_backbone.py#L197-L205
